### ISSUE TITLE
throttle metrics not resetting?

### ISSUE THREAD

**Issue by kfox1111** [ahmadalkhansa/vcgen-exporter#3]
I've seen a "arm frequency capped occured" and "soft temperature limit occured". Its constant 1 now, and I'm guessing it isn't actually that constant. Feels like it triggered and then is holding that value permanently?

If thats the case, how do you know when the issue is resolved?

**Comment by kfox1111** [comment]
Yeah. Looks like there are low bits and high bits. the low bits are "active" and the high ones are "occurred since the last reboot".

So, basically I need to kind of ignore the high bit ones.

**Comment by kfox1111** [comment]
Would it make sense to add a type=peristent|transient label on them (or something like that) so it can easily be filtered in alerts? If not, we can close the issue.

**Comment by ahmadalkhansa** [comment]
Hi, you can rely on the "arm frequency capped" and "soft temperature limit active" state labels for checking the current state. As you mentioned, these labels show the low bits. Adding a "type" label is not currently considered.
